### ISSUE TITLE
Allow now indicator arrow to "cross over" into the event space 

### ISSUE THREAD

**Issue by tylerturdenpants** [fullcalendar/fullcalendar#6537]
### Checklist

Please mark these items with an [x]

  - [x] I've already searched through [existing tickets](https://fullcalendar.io/issues)
  - [x] Other people will find this feature useful

### Feature Description

Allow now indicator arrow to "cross over" into the event space (sit in the middle of the division )

### Visual Mockups

![image](https://user-images.githubusercontent.com/18200519/132077987-07872712-d61f-49f4-ab6a-58517f402f87.png)

**Comment by PiotrGrzy** [comment]
@tylerturdenpants Have found any walkaround for time indicator across all days? Im interested for a solution in v5

**Comment by tylerturdenpants** [comment]
How do you mean, just the line across all days with only one dot like above or ....  could you mock something up so we hopefully mean the same thing?  I have found a work around for the above screen screenshot.

**Comment by PiotrGrzy** [comment]
Yes, exactly like on your screen, without the dot in my case, but thats not a problem.

**Comment by PiotrGrzy** [comment]
@tylerturdenpants Hey, can you provide me your solution?

**Comment by tylerturdenpants** [comment]
@PiotrGrzy Just to make sure of what you want, do you want to know how to make the now indicator line go from the very edge of the first day (before it would go into the time line) to very edge of the last day?
